orc8r/cloud/go/services/ctraced/servicers/trace_servicer_test.go

```diff
@@ -0,0 +1,101 @@
+/*
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package servicers_test
+
+import (
+	"testing"
+
+	"magma/orc8r/cloud/go/orc8r"
+	"magma/orc8r/cloud/go/serdes"
+	"magma/orc8r/cloud/go/services/configurator"
+	"magma/orc8r/cloud/go/services/configurator/test_init"
+	traceModels "magma/orc8r/cloud/go/services/ctraced/obsidian/models"
```

Comment:
General: snake_case these types of import aliases https://magma.github.io/magma/docs/next/contributing/contribute_conventions#go